sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_shared/models.py

```diff
@@ -6,210 +6,151 @@
 
 from enum import Enum, EnumMeta
 from six import with_metaclass
+from typing import Mapping, Optional, Union, Any
+try:
+    from typing import Protocol, TypedDict
+except ImportError:
+    from typing_extensions import Protocol, TypedDict
 
-import msrest
+from azure.core import CaseInsensitiveEnumMeta
```

Comment:
You will need to also bump the `azure-core` dependency to ensure `CaseInsensitiveEnumMeta` is present.